solutions/remotemonitoring/single-vm/setup.sh

```diff
@@ -95,6 +95,7 @@ install_docker_ce() {
     fi
 
     apt-get update -o Acquire::CompressionTypes::Order::=gz \
+        && apt-mark hold walinuxagent \
```

Comment:
Is walinuxagent only needed? Do we need to change it back?

Reply:
Yes only waLinuxAgent is supposed to be blacklisted. We are not trying to upgrading it anywhere so I'd think it's not necessary.